timeless/views.py

```diff
@@ -8,9 +8,13 @@ class UserView(AdminView):
 @todo #311:30min Once CreateView is implemented, refactor all blueprint views
```

Comment:
@paulodamaso this todo was already removed (in #310), so I'm not sure if we should leave it here or close it in the issue created from this puzzle (https://github.com/timelesslounge/timelessis/issues/443)?

Reply:
@krzyk Let's remove it, it was already resolved. I've opened an issue to @emilianodellacasa stating that we have too much overlapping and duplicated puzzles for views